packages/formation/sass/base/_b-variables.scss

```diff
@@ -87,6 +87,8 @@ $color-gold-light:           #f9c642;
 $color-visited:              $color-purple;
 $color-focus:                #3e94cf;
 
+$focus-outline:              2px solid $color-gold-light;
```

Comment:
By applying this variable override and the `@mixin focus` override, we don't need to have a whole file of overrides for the focus state.